src/elements/button/src/index.tsx

```diff
@@ -1,14 +1,43 @@
 /** @jsx jsx */
 
 import * as React from 'react'
-import { jsx } from 'theme-ui'
+import { jsx, useThemeUI } from 'theme-ui'
+import { darken } from '@theme-ui/color'
 
 export type Variant = 'primary' | 'secondary' | 'continue'
 type IconPosition = 'left' | 'center' | 'right' | null
 
 interface Props extends React.ButtonHTMLAttributes<HTMLButtonElement> {
   variant: Variant
   iconPosition?: IconPosition
+  inverse?: boolean
+}
+
+const invertTheme = (theme: any, variant: any = {}) => {
+  const backgroundColor = variant.color
+  const color = variant.backgroundColor || 'white'
+  const borderColor = color
+
+  const hoverColor = theme.colors[backgroundColor]
+    ? darken(theme.colors[backgroundColor], 0.1)()
+    : null
+
+  console.log({ variant })
+
+  return {
+    ...variant,
+    backgroundColor,
+    color,
+    borderColor,
+    borderWidth: 2,
+    borderStyle: 'solid',
```

Comment:
should we not add the variant after so it overrides

Reply:
@jonathan-fieldin I wanted to ensure that these took priority over the default variant.